### DISCUSSION TITLE
Include WebSocket connect, send, etc as flow node

### DISCUSSION THREAD

**Discussion by yoiang** [postmanlabs/postman-flows#244] [Ideas]
Hello!
Thanks so much for Postman, it's such a wonderful tool! 
I love the possible flexibility of the flow interface and was wondering fi there are plans to include WebSocket operations. There are times where I'd love to test a set of WebSocket message and responses, sometimes all in WebSockets, sometimes with some initial HTTP requests mixed in.

Cheers!

**Comment by saswatds** [comment]
Hey @yoiang we will slowly integrate all the different protocols that postman has started supporting. Websocket is indeed the next candidate.

**Reply by yoiang** [reply]
Awesome, thank you for the reply! I'm excited to get my hands on it!

**Comment by viko42** [comment]
Also looking for this integration :)
